test/run-broken.sh

```diff
@@ -15,7 +15,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -e
 set -o verbose
+bundle exec chef-zero &
+ZERO=$!
+trap "kill -9 $ZERO" EXIT
```

Comment:
Do we really want to `-9` here? This probably warrants a comment either way (same for the two other scripts).

Reply:
-9 seems right. we're exiting/being killed, we 100% want chef-zero to due immediately. Plus it's all in-memory so there's no danger.

Reply:
I'm really just mirroring what we have in [grocery-delivery](https://github.com/facebook/grocery-delivery/blob/master/test/run-git.sh) for these tests.